addons/l10n_ar/views/account_move_view.xml

```diff
@@ -30,6 +30,10 @@
                     </div>
                 </group>
             </group>
+
+            <xpath expr="//div[@name='journal_div']//field[@name='journal_id']" position="attributes">
+                 <attribute name="attrs">{'readonly': [('posted_before', '=', True)]}</attribute>
+            </xpath>
```

Comment:
This is far from ideal because it affects all the companies, not only the ones in Argentina.